README.md

```diff
@@ -15,6 +15,7 @@ For more information about this actuator and related repositories, see
 |---------------|---------------------|---------|
 | v1alpha2      | v1alpha2            | v0.2.X  |
 | v1alpha3      | v1alpha3            | v0.3.X  |
+| v1alpha4      | v1alpha3            | v0.4.X  |
```

Comment:
'Release', is that for CAPM3 or CAPI?

Reply:
@Jaakko-Os My understanding was CAPM3

Reply:
It would be nice to have unambiguous documentation. My proposal is to add CAPM3 to version field.

Reply:
@Jaakko-Os do you mean `Release field`? (i.e CAPM3 Release)

Reply:
yes, sorry for my typo

Reply:
Done, thanks!